worker/src/handlers/schedule.ts

```diff
@@ -5,7 +5,11 @@ import { average } from "../utils/average";
 import { formatDate } from "../utils/date";
 
 export async function handleSchedule(event: ScheduledEvent): Promise<void> {
-  const storedAnalytics = JSON.parse(await KV.get("core_analytics")) || {};
+  const storedAnalytics =
+    (await KV.get<{ [key: string]: CurrentAnalytics }>(
+      "core_analytics",
+      "json"
```

Comment:
The types and the docs are conflicting here, the docs say it should be `{type: "json"}` if you could verify this works, then I think we should keep it as we can keep the typing :-)

Reply:
Updated 9 days ago: <https://github.com/cloudflare/cloudflare-docs/pull/995>, but the typing way should also still work.

Reply:
`type: json` did not work when I tested that a few days ago.